modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseQueryServiceWaitStrategy.java

```diff
@@ -1,55 +1,17 @@
 package org.testcontainers.couchbase;
 
-import com.couchbase.client.core.message.cluster.GetClusterConfigRequest;
-import com.couchbase.client.core.message.cluster.GetClusterConfigResponse;
 import com.couchbase.client.core.service.ServiceType;
 import com.couchbase.client.java.Bucket;
 import lombok.extern.slf4j.Slf4j;
-import org.rnorth.ducttape.TimeoutException;
-import org.testcontainers.containers.ContainerLaunchException;
-import org.testcontainers.containers.wait.strategy.AbstractWaitStrategy;
-
-import java.time.Duration;
-import java.util.concurrent.TimeUnit;
-
-import static org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess;
 
 /**
- * @author ctayeb
- * Created on 06/06/2017
+ * @author tchlyah
```

Comment:
I don't think this should be touched (also, we don't generally use `@author` tags, just this one slipped in)

Reply:
Sorry, it was autogenerated. by Intellij, I'll revert it